dists/fs-repo-migrations/matrices/v1.0.0

```diff
@@ -0,0 +1,10 @@
+darwin 386
```

Comment:
(there is also the possibility of suppressing it from the distributions page, if we think "almost all users wont need it and it will be confusing". i twill still be available in the version file listing itself)